client/state/action-types.js

```diff
@@ -585,6 +585,9 @@ export const READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION = 'READER_UPDATE_NEW_POST
 export const RECEIPT_FETCH = 'RECEIPT_FETCH';
 export const RECEIPT_FETCH_COMPLETED = 'RECEIPT_FETCH_COMPLETED';
 export const RECEIPT_FETCH_FAILED = 'RECEIPT_FETCH_FAILED';
+export const REWIND_STATUS_ERROR = 'REWIND_STATUS_ERROR';
```

Comment:
this name might have room for improvement. since we're storing the error message it may be nice to indicate that this isn't a _failure_ of the request but a _positive_ indicator of a failure on the server. something like `REWIND_STATUS_REPORT_ERROR` or `REWIND_STATUS_UPDATE_ERROR`.

Further, why is this separate from `REWIND_STATUS_UPDATE`? I think I'm generally confused on the difference between a failed request to get the status and a request indicating a failure with rewind. If the server reports an error, will it return the `error` in a 2XX HTTP response?